### PR TITLE
fix (remove deprecated object.component)

### PR DESCRIPTION
**Pull request by tobiasschweizer**
This PR removes the deprecated `object.component` and replaces it with `resource.component` instead.

### CODE REVIEW

**Comment by tobiasschweizer** [comment]
@kilchenmann What about `model/services/deprecated/session.service.ts`? Can this be removed too?

**Comment by kilchenmann** [comment]
Yes the session.service is deprecated. You can remove it

**Comment by tobiasschweizer** [comment]
It is still used in `src/app/view/dashboard/user/user.component.spec.ts`. Could you take care of that?

**Comment by tobiasschweizer** [comment]
The best would be to make a branch from `develop` in a separate PR.

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -82,7 +82,7 @@ const appRoutes: Routes = [
     },
     {
         path: 'object/:rid',
```
Comment (tobiasschweizer) [inline src/app/app-routing.module.ts]: @kilchenmann Could you tell me what `object` means? Does it refer to the selector? Do I need to change it to `resource-object`?

**Pull request merged**